yarn-project/telemetry-client/src/metrics.ts

```diff
@@ -73,4 +79,90 @@ export const WORLD_STATE_SYNC_DURATION = 'aztec.world_state.sync.duration';
 export const WORLD_STATE_MERKLE_TREE_SIZE = 'aztec.world_state.merkle_tree_size';
 export const WORLD_STATE_DB_SIZE = 'aztec.world_state.db_size';
 
+export const WORLD_STATE_DB_MAP_SIZE_NULLIFIER = 'aztec.world_state.db_map_size.nullifier';
```

Comment:
Are all these constants used anywhere?

Reply:
Yes, they are used when we create the individual guages in code. They are then used in the grafana dashboard to reference the metrics.